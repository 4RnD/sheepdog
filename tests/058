#!/bin/bash

# Test discard/trim
seq=`basename $0`
echo "QA output created by $seq"

here=`pwd`
tmp=/tmp/$$
status=1        # failure is the default!

# get standard environment, filters and checks
. ./common.rc
. ./common.filter

_cleanup

for i in 0 1 2; do
	_start_sheep $i
done
_wait_for_sheep 3
$COLLIE cluster format
sleep 1
$COLLIE vdi create test 100M -P
$COLLIE node info | _filter_node_info
qemu-io -c "discard 0 100m" sheepdog:test | _filter_qemu_io
$COLLIE vdi check test
for i in `seq 0 24`; do
	$COLLIE vdi object test -i $i;
done
$COLLIE node info | _filter_node_info
