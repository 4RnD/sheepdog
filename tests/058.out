QA output created by 058
using backend plain store
Id	Size	Used	Use%
0 MASKED GB 104 MB MASKED
1 MASKED GB 104 MB MASKED
2 MASKED GB 104 MB MASKED
Total MASKED GB 312 MB MASKED

Total virtual image size	100 MB
discard 104857600/104857600 bytes at offset 0
100 MiB, X ops; XX:XX:XX.X (XXX YYY/sec and XXX ops/sec)
finish check&repair test
The inode object 0x7c2b25 idx 0 is not allocated
The inode object 0x7c2b25 idx 1 is not allocated
The inode object 0x7c2b25 idx 2 is not allocated
The inode object 0x7c2b25 idx 3 is not allocated
The inode object 0x7c2b25 idx 4 is not allocated
The inode object 0x7c2b25 idx 5 is not allocated
The inode object 0x7c2b25 idx 6 is not allocated
The inode object 0x7c2b25 idx 7 is not allocated
The inode object 0x7c2b25 idx 8 is not allocated
The inode object 0x7c2b25 idx 9 is not allocated
The inode object 0x7c2b25 idx 10 is not allocated
The inode object 0x7c2b25 idx 11 is not allocated
The inode object 0x7c2b25 idx 12 is not allocated
The inode object 0x7c2b25 idx 13 is not allocated
The inode object 0x7c2b25 idx 14 is not allocated
The inode object 0x7c2b25 idx 15 is not allocated
The inode object 0x7c2b25 idx 16 is not allocated
The inode object 0x7c2b25 idx 17 is not allocated
The inode object 0x7c2b25 idx 18 is not allocated
The inode object 0x7c2b25 idx 19 is not allocated
The inode object 0x7c2b25 idx 20 is not allocated
The inode object 0x7c2b25 idx 21 is not allocated
The inode object 0x7c2b25 idx 22 is not allocated
The inode object 0x7c2b25 idx 23 is not allocated
The inode object 0x7c2b25 idx 24 is not allocated
Id	Size	Used	Use%
0 MASKED GB 4.0 MB MASKED
1 MASKED GB 4.0 MB MASKED
2 MASKED GB 4.0 MB MASKED
Total MASKED GB 12 MB MASKED

Total virtual image size	100 MB
